website/content/en/highlights/2021-11-12-event-trottle-transform.md

```diff
@@ -0,0 +1,28 @@
+Event trottle transform released
+
+We've released a new transform that provides a user the ability to control the throughput of specific event streams.
+
+Large spikes in data volume can frequently overwhelm a service, which is especially common in log data. Previously, users lacked the necessary tooling to control throughput, such as setting quota for users and user groups utilizing Vector, which not only can cause spike in costs, but also negatively impact downstream services due to the increased load. 
+
+`Throttle` transform enables you to rate limit specific subsets of your event stream to limit load on downstream services or enforce quotas on users. You can utilize 'throttle' transform to enforce rate limits on number of events and exclude events based on a VRL condition to avoid dropping critical logs. 
+
+To ensure that each bucket's throughput averages out to the `threshold` per `window`, rate limiting spreads load across the configured `window`. The rate limiter will allow up to threshold number of events through and drop any further events for that particular bucket when the rate limiter is at capacity. 
+
+A rate limiter is created with a maximum number of cells equal to the `threshold`, with cells replenishing at a rate of `window` divided by `threshold`. To paint a better example, a `window` of 60 with a `threshold` of 10 replenishes a cell every 6 seconds and allows a burst of up to 10 events. 
```

Comment:
This would probably be great content to have in the `throttle` transform doc. Would you want to add this here:

https://github.com/vectordotdev/vector/blob/f74302c7c1c65bcdd6b7f2e91a478e9874d9385b/website/cue/reference/components/transforms/throttle.cue#L6-L8

as well?

Also the paragraph preceding the ones I highlighted.

Reply:
I believe that was taken from the docs I wrote: https://master.vector.dev/docs/reference/configuration/transforms/throttle/#rate-limiting

Reply:
Hah, indeed. Those are some good docs. I think this content could be made more discoverable by being moved up, but this is consistent with other component documentation for now.

Reply:
Yeah it seemed like description was much more of a small one liner

Reply:
Yeah, it is intended to be very concise. I just think we could surface more of the use-cases for components somehow. This is a broader thought on our documentation rather than anything specific about `throttle`.